bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/RetryingRpcFutureFallback.java

```diff
@@ -37,6 +37,12 @@
  */
 public class RetryingRpcFutureFallback<RequestT, ResponseT> implements FutureFallback<ResponseT> {
 
+  public static <RequestT, ResponseT> RetryingRpcFutureFallback<RequestT, ResponseT> create(
```

Comment:
You want to make the constructor private since you want to use static factory method?